src/Symfony/Bundle/TwigBundle/Resources/views/layout.html.twig

```diff
@@ -15,14 +15,14 @@
                 <h1 class="logo">{{ include('@Twig/images/symfony-logo.svg') }} Symfony Exception</h1>
 
                 <div class="help-link">
-                    <a href="https://symfony.com/doc">
+                    <a href="https://symfony.com/doc/{{ constant('Symfony\\Component\\HttpKernel\\Kernel::VERSION') }}/index.html" target="_blank">
                         <span class="icon">{{ include('@Twig/images/icon-book.svg') }}</span>
                         <span class="hidden-xs-down">Symfony</span> Docs
                     </a>
                 </div>
 
                 <div class="help-link">
-                    <a href="https://symfony.com/support">
+                    <a href="https://symfony.com/support" target="_blank">
```

Comment:
Not adding `_blank` was done on purpose. It's up to the user (here developers) to choose what to do; adding `_blank` forces the opening of a new tab/window. @pborreli can probably give a better explanation than me on this topic.

This should be reverted (same above).